tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+bender.amd.require( 'ckeditor', 'ckeditor-core' );
+
+describe( 'getPluginPath()', function( CKEDITOR, core ) {
```

Comment:
I agree, but please check my previous comment. This is intentional.

A generic solution may look different, in fact, but we should not miss the chance to bring an optimized solution to be used by us only.